apply-pr.js

```diff
@@ -41,13 +41,26 @@ if (!process.argv[2]) {
   exitWithMsg('Usage example: git-apply-pr joyent/node#1337');
 }
 
-var args = /(.*)\/(.*)#(\d*)/i.exec(process.argv[2]);
-if (!args) {
+var enablePlusOne = false;
```

Comment:
It changes when processing the command line arguments but it should not change elsewhere, hence why I described it as being _conceptually_ a constant.

But that's very minor, I don't have any strong opinion about it :)